src/components/ExperienceTimeline.tsx

```diff
@@ -1,23 +1,77 @@
 import { Box, Text } from "@chakra-ui/react";
-import { ExperienceDetail } from "./Experience";
-import styles from "./ExperienceTimeline.module.css";
+import { Experience } from "../consts";
 
 interface ExperienceTimelineProps {
-  experienceDetails: ExperienceDetail[];
-  updateExperienceDataIndex: (index: number) => void;
+  experienceDetails: Experience[];
+  setExperience: (experience: Experience) => void;
 }
 
+const getPseudoElementStyles = (isHovered: boolean) => {
+  let width = "0.313rem";
+  let height = "0.313rem";
+  let top = "0.625rem";
+  let left = "-0.05rem";
+  let boxShadow =
```

Comment:
We should not directly specify a color name but rather use theme-based global color constants declared in "consts" file.

Reply:
Yes. I would have done that in my next commit.